frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx

```diff
@@ -1,28 +1,30 @@
-# scExpression Documentation
+# scExpression — Query Gene Expression Across Tissues
+
+scExpression is a tool that allows to query the expression of any gene across all data in the cellxgene data portal. A query results in a dot plot per tissue as explained below.
```

Comment:
should we specify that its not actually querying all of the data in the data portal?

Reply:
you mean because of "is_primary_data"?

Reply:
that and the assay limits. And we only use datasets with 1 organism (although that only rules out 1 dataset I think)